Example/MoyaComparison/Environments/prod/Environment.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Environment.swift
+//  NetworkStackExample
+//
+//  Created by Steven_WATREMEZ on 12/3/17
+//  Copyright (c) 2017 niji. All rights reserved.
+//
+
+import Foundation
+
+enum Environment {
+  static let baseURL = "https://safe-retreat-51773.herokuapp.com"
```

Comment:
it's just for me to test my backend. So Dev is for localhost testing and Prod is for my heroku prod server. I will change that.